src/main/java/pages/SwipePage.java

```diff
@@ -12,9 +13,30 @@ public SwipePage() {
     @AndroidFindBy(xpath = "//android.view.ViewGroup[@content-desc='Swipe-screen']/android.view.ViewGroup[1]/android.widget.TextView")
     private MobileElement titleSwipe;
 
-    public String getTitleSwipeMenu(){
+    private MobileElement elementCarousel;
+
+    private String horizontalScrollView = "new UiSelector().className(\"android.widget.HorizontalScrollView\")";
+
+    private String neededElement_finder = "new UiSelector().text(\"%s\")";
+
+    public String getTitleSwipeMenu() {
         wait.untilElementIsVisible(timeOutSeconds, titleSwipe);
         return titleSwipe.getText();
     }
 
+    public void selectCarruselOption(String text) {
+        switch (dotenv.get("PLATFORM_NAME")) {
+            case "iOS":
+                //Aquí va lo relacionado a la busqueda en iOS
```

Comment:
Does this iOS implementation will eventually be done?

Reply:
Yes, that is the idea